eng/common/scripts/modules/ChangeLog-Operations.psm1

```diff
@@ -94,8 +94,7 @@ function Confirm-ChangeLogEntry {
   }
 
   if ($ForRelease -eq $True) {
-    $CurrentDate = Get-Date -Format "yyyy-MM-dd"
-    if ($changeLogEntry.ReleaseStatus -ne "($CurrentDate)") {
+    if ($changeLogEntry.ReleaseStatus -eq "(Unreleased)") {
       Write-Host ("##[warning]Incorrect Date: Please use the current date in the Changelog '{0}' before releasing the package" -f $ChangeLogLocation)
```

Comment:
Please update the error message as well:

` Write-Host ("##[warning]No release date set. Please set a release date with format 'yyyy-MM-dd' in the heading for version '{0}' in the changelog '{1}'." -f $VersionString $ChangelogLocation)`